VMs/metrics/README.md

```diff
@@ -0,0 +1,242 @@
+# Kata Containers metrics Jenkins CI VMs
+
+This directory contains a set of configuration files and helper scripts
+to aid in setting up a [Jenkins](https://jenkins.io/) CI system to run
+the [Kata Containers metrics tests](https://github.com/kata-containers/tests/tree/master/metrics)
+inside Virtual Machines (VMs).
```

Comment:
I wonder if it's worth adding a link to http://kata-jenkins-ci.westus2.cloudapp.azure.com?

Reply:
Hmm. Maybe, but that is not where these VM images run - that uses, afaik, the Azure base images with some `cloud-init` on top that we store [in our Jenkins xml configs](https://github.com/kata-containers/ci/blob/master/jenkins/config.xml#L192).
Where you thinking of a new section or something to discuss the Azure setup vs the bare-metal setup?

Reply:
I'm easy. This was just an idea after I noticed that none of the doc in this CI mentions the location of where you can view the the CI's built from this repo.